lang/en.json

```diff
@@ -87,5 +87,9 @@
   "{1} is loading.": "{1} is loading.",
   "Exit Picture-in-Picture": "Exit Picture-in-Picture",
   "Picture-in-Picture": "Picture-in-Picture",
-  "No content": "No content"
+  "No content": "No content",
+  "Color": "Color",
```

Comment:
We're going to need an en-GB :)

Reply:
Good call, created this issue here: https://github.com/videojs/video.js/issues/8105

Reply:
I think mastodon using british naming balances things out 😉